packages/dnb-eufemia/src/shared/locales/en-GB.js

```diff
@@ -43,15 +43,15 @@ export default {
     },
     GlobalError: {
       404: {
-        title: "Oops! We can't find the page you're looking for …",
-        text: 'Did we messed with the links? Try again, or [go back where you came from](/back).',
-        alt: 'Lady searching in empty box',
+        title: "We can't find the page you're looking for …",
+        text: 'Are you sure you have entered the correct address? Or have we messed with the links?',
       },
       500: {
-        title: 'Ohh, a technical error happened!',
+        title: 'Sorry, a technical error happened!',
         text: 'The service is not working properly at the moment, but try again later.',
```

Comment:
```suggestion
        text: 'The service is not working properly at the moment. Try again later.',
```

I'm no text author, but I felt that the original text ", but try again later" sounded a bit strange.